### PR TITLE
tornado -> requests

### PR DESCRIPTION
**Pull request by shawnzhu**
Supports #37

After this change, my following example works:

```Python
import os

import ibmcloudsql
import pandas as pd

API_KEY = os.environ.get('IBMCLOUD_API_KEY')
INSTANCE_CRN = os.environ.get('SQLQUERY_INSTANCE_CRN')
TARGET_COS_URL = 'cos://us-south/xxx/result/'

sqlClient = ibmcloudsql.SQLQuery(API_KEY, INSTANCE_CRN, target_cos_url=TARGET_COS_URL)
sqlClient.logon()

sql = 'SELECT * FROM cos://us-south/xxx/yyy/ WHERE dt = \'2019-06-02\' LIMIT 5'

df = sqlClient.run_sql(sql)

print(df)
```

## Goal 

I would prefer to complete experiment on branch `experiment-requests` then try to merge with validation by an integration test case like :point_up example.

### CODE REVIEW

File: Python/ibmcloudsql/SQLQuery.py [hunk 1]
```diff
@@ -344,13 +337,12 @@ def delete_result(self, jobId):
 
         fourth_slash = result_location.replace('/', 'X', 3).find('/')
 
-        response = self.client.fetch(
+        response = requests.get(
             result_location[:fourth_slash] + '?prefix=' + result_location[fourth_slash + 1:],
-            method='GET',
-            headers=self.request_headers,
-            validate_cert=False)
+            params=self.request_headers,
```
Comment (emmadickson) [inline Python/ibmcloudsql/SQLQuery.py]: Should this end with a , after params?

File: Python/ibmcloudsql/SQLQuery.py [hunk 2]
```diff
@@ -296,13 +290,12 @@ def list_results(self, jobId):
...
```
Comment (emmadickson) [inline Python/ibmcloudsql/SQLQuery.py]: Should this end with a , after params?

File: Python/ibmcloudsql/SQLQuery.py [hunk 3]
```diff
@@ -387,43 +379,40 @@ def get_job(self, jobId):
             return
 
         try:
-            response = self.client.fetch(
+            response = requests.get(
                 "https://sql-api.ng.bluemix.net/v2/sql_jobs/{}?instance_crn={}".format(jobId, self.instance_crn),
-                method='GET',
                 headers=self.request_headers,
```
Comment (emmadickson) [inline Python/ibmcloudsql/SQLQuery.py]: Should this end with a , after headers?

File: Python/ibmcloudsql/SQLQuery.py (hunk 2 above)
Comment (shawnzhu) [reply Python/ibmcloudsql/SQLQuery.py]: Good question. short answer is yes, trailing comma is recommended for multi-line parameters. See [When to Use Trailing Commas](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)

and practice https://stackoverflow.com/a/11597911/914967

> The main advantages are that it makes multi-line lists easier to edit and that it reduces clutter in diffs.

**Pull request merged**